lib/vat_info/models/status.rb

```diff
@@ -0,0 +1,25 @@
+module VatInfo
+  module Models
+    class Status
+      attr_accessor :data
+
+      def initialize(params = {})
+        @params        = params
+        @data          = {}
+        @data[:status] = create_status
+      end
+
+      def create_status
+        {
+          status_code: @params.fetch(:@status_code),
```

Comment:
Zkus se zamyslet nad tím jestli by nebylo lepší tu sem dát místo toho jejich debilního číslování ( se zas překonali s tim API ) třeba nějaký symbol jakože :ok, nebo :too_many_dic_subjects ... podle toho co ty jejich kódy znamenaj.

Reply:
a tak teď na to koukám a on je ten důvod asi v tom status_text je to tak? Pokud ano tak předchozí komentář ignoruj :)

Reply:
A používej tu ty attr_accessor místo @ ( u params ).

Reply:
attr_accessors fixed